packages/volto/src/components/manage/Sidebar/Sidebar.jsx

```diff
@@ -186,19 +192,16 @@ const Sidebar = (props) => {
             },
             !!orderTab && {
               menuItem: intl.formatMessage(messages.order),
-              pane: (
-                <Tab.Pane
-                  key="order"
-                  className="tab-wrapper"
-                  id="sidebar-order"
-                >
-                  <Icon
-                    className="tab-forbidden"
-                    name={forbiddenSVG}
-                    size="48px"
-                  />
-                </Tab.Pane>
-              ),
+              pane:
+                isOrderTabRendered || tab === 2 ? (
```

Comment:
If my assumptions above are correct, this can simply be
```suggestion
                tab === 2 ? (
```